datasets/dane/dane.py

```diff
@@ -0,0 +1,283 @@
+      Barrett, Maria  and
+      Rosted, Christina  and
+      Lidegaard, Lasse Malm  and
+      S{\o}gaard, Anders",
+    booktitle = "Proceedings of the 12th Language Resources and Evaluation Conference",
+    month = may,
+    year = "2020",
+    address = "Marseille, France",
+    publisher = "European Language Resources Association",
+    url = "https://www.aclweb.org/anthology/2020.lrec-1.565",
+    pages = "4597--4604",
+    abstract = "We present a named entity annotation for the Danish Universal Dependencies treebank using the CoNLL-2003 annotation scheme: DaNE. It is the largest publicly available, Danish named entity gold annotation. We evaluate the quality of our annotations intrinsically by double annotating the entire treebank and extrinsically by comparing our annotations to a recently released named entity annotation of the validation and test sections of the Danish Universal Dependencies treebank. We benchmark the new resource by training and evaluating competitive architectures for supervised named entity recognition (NER), including FLAIR, monolingual (Danish) BERT and multilingual BERT. We explore cross-lingual transfer in multilingual BERT from five related languages in zero-shot and direct transfer setups, and we show that even with our modestly-sized training set, we improve Danish NER over a recent cross-lingual approach, as well as over zero-shot transfer from five related languages. Using multilingual BERT, we achieve higher performance by fine-tuning on both DaNE and a larger Bokm{\aa}l (Norwegian) training set compared to only using DaNE. However, the highest performance isachieved by using a Danish BERT fine-tuned on DaNE. Our dataset enables improvements and applicability for Danish NER beyond cross-lingual methods. We employ a thorough error analysis of the predictions of the best models for seen and unseen entities, as well as their robustness on un-capitalized text. The annotated dataset and all the trained models are made publicly available.",
+    language = "English",
+    ISBN = "979-10-95546-34-4",
+}
+"""
+
+_DESCRIPTION = """\
+The DaNE dataset has been annotated with Named Entities for PER, ORG and LOC
+by the Alexandra Institute.
+It is a reannotation of the UD-DDT (Universal Dependency - Danish Dependency Treebank)
+which has annotations for dependency parsing and part-of-speech (POS) tagging.
+The Danish UD treebank (Johannsen et al., 2015, UD-DDT) is a conversion of
+the Danish Dependency Treebank (Buch-Kromann et al. 2003) based on texts
+from Parole (Britt, 1998).
+"""
+
+_HOMEPAGE = "https://github.com/alexandrainst/danlp/blob/master/docs/docs/datasets.md#dane"
+
+_LICENSE = "CC BY-SA 4.0"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://danlp.alexandra.dk/304bd159d5de/datasets/ddt.zip"
+
+
+class DaNE(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Dane(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `dane`